src/XliffTasks/build/XliffTasks.targets

```diff
@@ -165,4 +166,22 @@
     <TranslateSource XlfFile="@(Xlf)" />
   </Target>
 
+  <!--
+    Sorts the trans-unit elements in the .xlf files by the value of their id attributes.
+  -->
+  <Target Name="SortXlf"
+          DependsOnTargets="GetXlfSources;UpdateXlfSourceList"
+          Inputs="@(XlfSource);$(_XlfSourceList)"
+          Outputs="$(_UpdateXlfLastRun)"
```

Comment:
Not sure about these inputs/outputs. It doesn't re-update the XLF from source, it just sorts existing nodes, right? This will make UpdateXlf think its up-to-date if SortXlf was run. I think SortXlf can be non-incremental since it's a manual operation and not an inner loop one.

Reply:
Err, sorry. Meant to replace `_UpdateXlfLastRun` with a new property, `_SortXlfLastRun`. Would that take care of your concern?

Reply:
It would, but maybe not worth it. See #28 which this would also suffer from and potentially need fixing. Since Sort is a one-off, I think it's ok to do it without this incremental optimization.

Reply:
Ah, I see. In that case I can just drop the `Inputs` and `Outputs` as well?

Reply:
Here, yes.